File: src/AoC16/s16e01-csharp/Part2.cs
using System;

namespace AoC16S16E01
{
    class Part2
    {
        static void Main()
        {
            string input = Console.In.ReadToEnd().Trim();
            int result = Solution.SolvePart2(input);
            Console.WriteLine(result);
        }
    }
}

File: src/AoC25/s25e10-cpp/src/part2.cpp
#include <iostream>
#include <sstream>

#include "solution.h"

std::string read_stdin() {
    std::ostringstream buffer;
    buffer << std::cin.rdbuf();
    return buffer.str();
}

int main() {
    std::string input = read_stdin();
    int64_t result = aoc::solve_part2(input);
    std::cout << result << "\n";
    return 0;
}
